akka-docs-new/src/main/paradox/guide/quickstart.md

```diff
@@ -62,52 +62,61 @@ organization := "intro-akka.organization"
 version := "0.1-SNAPSHOT"
 
 scalaVersion := "2.11.8"
-val AkkaVersion = "2.4.12"
+val AkkaVersion = "2.4.17"
 
 libraryDependencies += "com.typesafe.akka" %% "akka-actor" % AkkaVersion
 libraryDependencies += "com.typesafe.akka" %% "akka-testkit" % AkkaVersion % "test"
 ```
 
-This simple file sets up first the project metadata (_name_ and _organization_; we just picked a sample one here). 
-Then we set up the version of the scala compiler we use, then set a variable with the Akka version we intend to
-use (always try to use the latest). 
+This simple file sets up first the project metadata ( _name_ and _organization_; we just picked a sample one here).
+Thereafter we set up the version of the Scala compiler we use, then set a variable with the Akka version we intend to
+use. Always strive to use the latest version.
 
-As the last step, we add two dependencies, `akka-actor` and `akka-testkit`. Note that we used the `AkkaVersion` 
+As the last step, we add two dependencies, `akka-actor` and `akka-testkit`. Note that we used the `AkkaVersion`
 variable for both dependencies, ensuring that versions are not accidentally mixed and kept in sync when upgrading.
 
-Finally, check that everything works by running `sbt update` from the base directory of your project 
+Finally, check that everything works by running `sbt update` from the base directory of your project
 (where the `build.sbt` file is).
 
-## Setting up your IDE
+## Setting up Your IDE
```

Comment:
Yes, I think so. Here's the guide I used to determine:
http://grammar.yourdictionary.com/capitalization/rules-for-capitalization-in-titles.html